ethpm/exceptions.py

```diff
@@ -3,16 +3,18 @@ class InsufficientAssetsError(Exception):
     Error to signal a Manifest or Package does not
     contain the required assets to do something.
     """
+    pass
 
 
-class UriNotSupportedError(Exception):
+class ValidationError(Exception):
     """
-    Error to signal a URI scheme is not supported.
+    Error to signal something does not pass a validation check.
```

Comment:
I would suggest to adopt the `Raised when` scheme for exception doc strings that we adopted from the Python standard lib for Py-EVM/Trinity.

![image](https://user-images.githubusercontent.com/521109/41748351-c9776cfa-75b1-11e8-908c-994ae9eaf6eb.png)

Reply:
Dope, thanks for the tip!